tests/unit/events/test_logging.py

```diff
@@ -1,7 +1,10 @@
+from argparse import Namespace
```

Comment:
Reverting this commit fixes our broken `tests/unit/graph/test_selector.py::test_run_specs[a,b--]` test problem. I don't understand why.